forge-gui/src/main/java/forge/gamemodes/match/input/InputSelectEntitiesFromList.java

```diff
@@ -130,14 +130,21 @@ protected String getMessage() {
                 ? String.format(message, selected.size())
                         : String.format(message, max - selected.size()));
 
-        if (sa != null && sa.hasParam("Crew") && sa.getPayCosts().hasSpecificCostType(CostTapType.class)) {
-            msg.append("\nCrewing: ").
-            append(CardLists.getTotalPower((FCollection<Card>)getSelected(), true, true)).
-            append(" / ").append(TextUtil.fastReplace(sa.getPayCosts().getCostPartByType(CostTapType.class).getType(), "Creature.Other+withTotalPowerGE", ""));
-        } else if (sa != null && sa.getPayCosts().hasSpecificCostType(CostExile.class) && tally > 0) {
-            msg.append("\n").append(Localizer.getInstance().getMessage("lblCMC")).append(": ");
-            msg.append(CardLists.getTotalCMC((FCollection<Card>)getSelected())).append(" / ").append(tally);
+        if (sa != null) {
+            if (sa.getPayCosts().hasSpecificCostType(CostTapType.class) &&
+                (sa.hasParam("Crew") || sa.hasParam("Saddle"))) {
```

Comment:
try this instead: `sa.isKeyword(Keyword.SADDLE)`

i haven't removed "Crew" yet because of _Heart of Kiran_ 's ability